Yoast/wordpress-seo
# Merge company or person and social profiles in onboarding wizard

## Summary

This PR can be summarized in the following changelog entry:

* Merged company or person and social profiles in onboarding wizard.

## Relevant technical choices:

* Added `react-select` for selecting a WordPress user.

## Test instructions

Designs followed: https://github.com/Yoast/design/issues/315

This PR can be tested by following these steps:

* If https://github.com/Yoast/javascript/pull/184 hasn't been merged yet, make sure to link the mono repo this that PR.

## UI changes

## Quality assurance


Requires https://github.com/Yoast/javascript/pull/184
Requires https://github.com/Yoast/javascript/pull/193

Fixes #12516 
Fixes #12519
